src/components/header/style.module.css

```diff
@@ -29,7 +31,7 @@
 	.open & {
 		justify-content: space-between;
 		-webkit-overflow-scrolling: touch;
-		height: var(--vh);
+		height: calc(var(--vh) - var(--banner-height));
```

Comment:
Predates my involvement, but IIUC, this was a problem introduce when the banner was added. It seems like the idea was that the social icons would be at the bottom of the screen when the togglable header was expanded, however, it never took into account the banner height. As such, the icons were always clipped off the screen.

The handful of white pixels at the very bottom of the screen are our icons, but you obviously can't interact with them as-is.

![Screenshot of the site on a mobile device with the header expanded. Shows just the very top of icons (a handful of pixels at most)](https://github.com/user-attachments/assets/3568f04f-0d33-4a0a-9469-5305b2ff1fba)